pkg/reconciler/autoscaling/kpa/kpa.go

```diff
@@ -252,16 +252,17 @@ func computeActiveCondition(pa *pav1alpha1.PodAutoscaler, want int32, got int) {
 			pa.Status.MarkInactive("NoTraffic", "The target is not receiving traffic.")
 		}
 
-	case got < minReady && want > 0:
-		pa.Status.MarkActivating(
-			"Queued", "Requests to the target are being buffered as resources are provisioned.")
+	case got < minReady:
+		if want > 0 || !pa.Status.IsInactive() {
```

Comment:
What is `!pa.Status.IsInactive()` guarding against?  Can we just to this unconditionally?

Same below.

Reply:
My initial commit was doing it unconditionally, but as @vagababov pointed out, [that will do the wrong thing](https://github.com/knative/serving/pull/4928#issuecomment-515121205) in some [cases](https://github.com/knative/serving/pull/4928#issuecomment-515251838). 

In both cases, the guard will prevent marking an Inactive PA as "activating"/"active" while it is presumably scaling down.

Reply:
That table is great.  Any way we can work it into a detailed comment? 👼

Reply:
Sounds good